src/providers/Celo.ts

```diff
@@ -40,6 +45,8 @@ const ERC20_SUBSET = [
 const kit = newKit(process.env.CELO_NODE_RPC_URL)
 const curveBalanceCalculator = CurvePoolBalanceCalculator.Instance
 const uniV3BalanceCalculator = UniV3PoolBalanceCalculator.Instance
+const provider = new JsonRpcProvider(process.env.CELO_NODE_RPC_URL)
```

Comment:
do we have any defense mechanism to check for presence of env vars in this repo? i don't see anything so it's not particular to this PR but in general it's safer to always check env vars before using them either inline via `if (!process.env.SOMETHING) throw new Error('Missing env var SOMETHING')` or using a package like https://www.npmjs.com/package/env-var

no need to fix in this PR, just a general comment